index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="./about.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header id="About">
+      <nav class="nav-bar mobile-header">
+        <div class="mobile-logo"></div>
+        <div class="openMenu"><i class="fa fa-bars"></i></div>
+        <ul class="mainMenu">
+          <li class=""><a href="./about.html" class="nav-link">About</a></li>
+          <li class=""><a href="#" class="nav-link">Programs</a></li>
+          <li class=""><a href="#" class="nav-link">Join</a></li>
+          <li class=""><a href="#" class="nav-link">Sponsor</a></li>
+          <li class=""><a href="#" class="nav-link">News</a></li>
+          <li class=""><a href="#" class="nav-link">WHO</a></li>
+          <div class="closeMenu"><i class="fa fa-times"></i></div>
+        </ul>
+      </nav>
+      <nav class="desktop-header">
+        <div class="desktop-first-nav">
+          <li class="">
+            <a href="#" class="nav-link"><img src="./icons/github.png" alt="" /></a>
+          </li>
+          <li class="">
+            <a href="#" class="nav-link"><img src="./icons/twitter.png" alt="" /></a>
+          </li>
+          <li class=""><a href="#" class="nav-link">Logout</a></li>
+          <li class=""><a href="#" class="nav-link">Mypage</a></li>
+          <li class=""><a href="#" class="nav-link">Korean</a></li>
+        </div>
+        <div class="desktop-second-nav">
+          <div class="desktop-logo">
+            <a href="./index.html"><img src="./icons/logo22.png" alt="" /></a>
```

Comment:
- [ ] Great job using images in your HTML files! :+1: However, please note that it is required to always add a descriptive `alt` attribute to each image.
The `alt` attribute provides alternative information for the image in case it cannot be displayed or accessed by a user. This is particularly important for users who may be visually impaired or using assistive technologies to navigate your website.
By providing descriptive `alt` attributes for each image, you can ensure that your website is accessible to as many users as possible and meets accessibility standards. Kindly check **all** the images inside your project :+1: